### DISCUSSION TITLE
Hi ,

### DISCUSSION THREAD

**Discussion by Hghadjari** [gimli-org/pyGIMLi#682] [Q&A]
I need to create a rectangular grid (structured mesh) and get the values of the potential at the grid points. The function "simulate" accepts "mesh" as the first argument. The mesh is created by "createMesh" which needs the "geom" as the first argument. Our problem is that the "createMesh" produces triangular cells for finite element, however, we need rectangular cells. The function "createGrid" does not accept the geometry as the argument.

In the Tutoiral in the Synthetic data generation:
data = ert.simulate(mesh, scheme=scheme, res=rhomap, noiseLevel=1,
noiseAbs=1e-6, seed=1337)
ert.simulate accepts mesh.

**Comment by halbmy** [comment]
Of course, `createGrid` does not accept an arbitrary geometry as triangle cells can. It just accepts two (or three in 3D) vectors for the discretization. Please search for createGrid and have a look at the examples.
